keyboards/takashicompany/goat51/info.json

```
@@ -0,0 +1,499 @@
+{
+    "manufacturer": "takashicompany",
+    "keyboard_name": "GOAT51",
+    "maintainer": "takashicompany",
+    "bootloader": "caterina",
```

If you set the dev board, it sets the bootloader and processor type. 

```suggestion
    "development_board": "promicro",
```